.eslintrc

```diff
@@ -370,6 +372,22 @@
       "rules": {
         "no-restricted-imports": "off"
       }
+    },
+    {
+      "files": ["src/app/i18next-t.ts"],
```

Comment:
It would err in the `i18next-t.ts` file after fixing it, so I added an override here. It might be heavy handed though so open to deleting it based on your preference.

Reply:
Probably easier to just ignore the one error?

Reply:
Yeah I considered that too, will raise a cleanup pr

Reply:
I got it